packages/astro/src/vite-plugin-mdx/index.ts

```diff
@@ -0,0 +1,134 @@
+import type { TransformResult } from 'rollup';
+import {
+	transformWithEsbuild,
+	type Plugin,
+	type ResolvedConfig,
+} from 'vite';
+import type { AstroRenderer, AstroSettings } from '../@types/astro';
+import type { LogOptions } from '../core/logger/core.js';
+import type { PluginMetadata } from '../vite-plugin-astro/types';
+
+import babel from '@babel/core';
```

Comment:
Any chance we can drop `@babel/core` if this is the only place we're using it?

Reply:
In favor of esbuild you mean?

Reply:
esbuild already runs. I'm not sure what implications there are if we remove this code to be honest.